membership-common/src/test/scala/com/gu/memsub/subsv2/services/SubscriptionServiceTest.scala

```diff
@@ -341,13 +314,6 @@ class SubscriptionServiceTest extends Specification {
       result mustEqual \/-("idPartner")
     }
 
-    "tell you you aren't a staff membership if your subscription is cancelled regardless of the date" in {
```

Comment:
no such thing as instant change of a one time charge subscription any more.  we only modelled friend and staff that way.